php-cs-fixer

```diff
@@ -57,11 +57,13 @@ if ($require) {
     // OK, it's not, let give Composer autoloader a try!
     if (file_exists($a = __DIR__.'/../../autoload.php')) {
         require_once $a;
+    } elseif (file_exists($b = __DIR__.'/../autoload.php')) {
```

Comment:
Can we change to `foreach` like in [here](https://github.com/sebastianbergmann/phpunit/blob/8.2.5/phpunit#L30)?